### PR TITLE
NO-ISSUE: Change assisted presubmit jobs conformance tests to minimal

### PR DESCRIPTION
**Pull request by danmanor**
Change conformance tests in `presubmit` `assisted-baremetal` jobs to run minimal list of conformance tests

### CODE REVIEW

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

File: ci-operator/step-registry/assisted/baremetal/test/assisted-baremetal-test-commands.sh [hunk 1]
```diff
@@ -31,10 +31,17 @@ test_list_file="test-list"
 test_skips_file="test-skips"
 test_env_file="test-env"
 
+if [ ${TEST_TYPE:-""} == "minimal" ]; then
+    echo "Running minimal test list"
+    read -d '#' TEST_LIST << EOF
+[sig-cli] Kubectl client Kubectl api-versions should check if v1 is in available api versions
```
Comment (adriengentil) [inline ci-operator/step-registry/assisted/baremetal/test/assisted-baremetal-test-commands.sh:37]:
```suggestion
"[sig-cli] Kubectl client Kubectl api-versions should check if v1 is in available api versions [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
```
let's put it as a variable in the the workflow. Also, this string won't match any tests.

If it's a variable, I wonder if we can set `TEST_LIST: ${TEST_LIST_MINIMAL}` in job definition, so we won't have to define a new test type.

Comment (danmanor) [reply ci-operator/step-registry/assisted/baremetal/test/assisted-baremetal-test-commands.sh:37]: I guess we can, I will try

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

File: ci-operator/step-registry/assisted/baremetal/test/assisted-baremetal-test-ref.yaml [hunk 2]
```diff
@@ -15,17 +15,26 @@ ref:
...
     documentation: If the value is equal to "true", run only the upgrade tests
   - name: TEST_TYPE
-    default: "list"
+    default: fixed
     documentation: |-
       Specifies the type of test to run:
       * 'suite' - Run a full test suite. If chosen, 'TEST_SUITE' and 'TEST_PROVIDER' can be specified.
       * 'list' - Run a specific list of tests. 'TEST_LIST' specified.
+      * 'fixed' - Run a fixed test list.
       * 'none' - Don't run any tests.
```
Comment (adriengentil) [inline ci-operator/step-registry/assisted/baremetal/test/assisted-baremetal-test-ref.yaml:24]:
```suggestion
      * 'custom' - Run a specific list of tests. 'TEST_LIST' must be specified.
      * 'minimal' - Run the minimal test list.
      * 'extensive' - Run the extensive test list.
      * 'none' - Don't run any tests.
```

and let's remove `FIXED_TEST_LIST` as it looks redondant

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-periodic pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted

**Pull request merged**